src/database/schema.sql

```diff
@@ -133,7 +133,9 @@ CREATE TABLE IF NOT EXISTS endpoints (
     documentation_object UUID REFERENCES objects(id) ON DELETE CASCADE,
```

Comment:
Was not changed here, but this should not be `ON DELETE CASCADE` otherwise deleting the object would result in a deletion of the endpoint which could be quite dangerous